apps/web/components/App.tsx

```diff
@@ -94,7 +95,7 @@ export default function App({
             </Link>
             <div className="items-center justify-between py-4 sm:flex sm:py-8">
               <div className="flex">
-                <img className="h-16 w-16" src={logo} alt={name} />
+                <Image width={64} height={64} src={logo} alt={name} layout="fixed" />
```

Comment:
We really don't need to optimize these logos. Also let's take into account that these have some tradeoff. Each call is adding up to our Vercel usage.